test/e2e/config.go

```diff
@@ -2,7 +2,7 @@ package integration
 
 var (
 	redis             = "quay.io/libpod/redis:alpine"
-	fedoraMinimal     = "quay.io/libpod/fedora-minimal:latest"
+	fedoraMinimal     = "registry.fedoraproject.org/fedora-minimal:34"
```

Comment:
What makes me uncomfortable here is that `registry.fp.org` has a long and glorious tradition of flaking. If we stick with `quay`, at least it's all-or-nothing.